docs/source/serving/sky-serve.rst

```diff
@@ -74,9 +74,14 @@ Use :code:`sky serve status` to check the status of the service:
 
    <div style="height: 20px;"></div>
 
+.. tip::
+
+  Notice that the two replicas are launched in different regions/clouds for the lowest cost and highest GPU availability.
+  This is performed automatically, like a regular ``sky launch``.
```

Comment:
```suggestion
  This is performed automatically on all enabled regions/clouds specified in the task YAML, like a regular ``sky launch``.
```
not sure; cloud discuss.

Reply:
I think at this point in the tutorial (Quick tour), we haven't said much about the yaml, and we want to be brief here.

Unless you think the current wording is too ambiguous?

Reply:
Good point! Let's skip it.